c2corg_ui/static/js/documentediting.js

```diff
@@ -151,6 +151,12 @@ app.DocumentEditingController = function($scope, $element, $attrs, appLang,
   this.submit = false;
 
   /**
+   * @type {Date}
+   * @export
+   */
+  this.today = new Date();
```

Comment:
writing for example `ng-init="today = new Date"(`) will not work.

Maybe we could use a mako variable and render it with pyton `today = ${today}` ? 
why is it more appropriate to set it in the template rather than here?